nilearn/decoding/searchlight.py

```diff
@@ -273,6 +281,10 @@ def fit(self, imgs, y):
             Target variable to predict. Must have exactly as many elements as
             3D images in img.
 
+        groups : array-like, optional
+            group membership for each sample for cross validation. Must have
+            exactly as many elements as 3D images in img. default None
+
```

Comment:
same here adding NOTE to say users that this argument only available for scikit learn >= 0.18